src/Plugin/Create/Payload.php

```diff
@@ -22,6 +22,8 @@
  */
 final class Payload extends BasePayload
 {
+	public static ?string $requestTarget;
```

Comment:
Removing the nullable declaration for `$requestTarget` and using just `string` is a reasonable improvement. This makes the code more straightforward and avoids the need for a `match` block to handle a potentially null value.